activity_calendar/feeds.py

```diff
@@ -74,11 +89,19 @@ def vtimezone(self):
 
     def items(self):
         # Only consider published activities
-        return Activity.objects.filter(published_date__lte=timezone.now()).order_by('-published_date')
+        activities = Activity.objects.filter(published_date__lte=timezone.now()).order_by('-published_date')
+        exceptions = ActivityMoment.objects.all()
```

Comment:
I don't think we should include activity moments for which their parent activity is not published yet.